lib/compress.js

```diff
@@ -1739,22 +1739,22 @@ merge(Compressor.prototype, {
             "valueOf",
         ];
         var native_fns = {
-            Array: makePredicate([
+            Array: [
                 "indexOf",
                 "join",
                 "lastIndexOf",
                 "slice",
-            ].concat(object_fns)),
-            Boolean: makePredicate(object_fns),
-            Number: makePredicate([
+            ].concat(object_fns),
```

Comment:
This section is for expressions like `"abc".charCodeAt(1)`, and all of these types inherit the same three functions from `Object`:
```js
"constructor"
"toString"
"valueOf"
```
Do you want me to be verbose and put those values into each of those keys instead?

Note that this only applies to `native_fns` and not `static_fns`, i.e. we don't have `Array.constructor`